phpunit.xml.dist

```diff
@@ -14,11 +14,9 @@
         </testsuite>
     </testsuites>
 
-    <!--
     <php>
-        <server name="KERNEL_DIR" value="/path/to/your/app/" />
+        <server name="KERNEL_DIR" value="/app/" />
     </php>
-    -->
 
     <filter>
         <whitelist>
```

Comment:
the paths in the rest if the file are wrong. it lists `../src` intead of just `src`